package/upload_binary.py

```diff
@@ -0,0 +1,91 @@
+"""Uploads the PyInstaller binary to our S3 bucket for storing releases, then creates
+a CloudFront invalidation to make those changes live.
+"""
+
+import argparse
+import os
+import sys
+import uuid
+from io import BytesIO
+from pathlib import Path
+
+import boto3
+from brainframe.cli import version_tag
+
+ssm = boto3.client("ssm")
+s3 = boto3.client("s3")
+cloudfront = boto3.client("cloudfront")
+
+
+_COMPANY_NAMES = ["aotu", "dilililabs"]
+
+
+def main():
+    stage = os.environ.get("STAGE", "dev")
+
+    args = _parse_args()
+
+    if not args.binary_path.exists():
+        print(
+            f"Missing binary at '{args.binary_path}'. Has a build been run?",
+            file=sys.stderr,
+        )
```

Comment:
```suggestion
        message = f"Missing binary at '{args.binary_path}'. Has a build been run?"
        print(message, file=sys.stderr)
```

Reply:
Are we not using 88 char lines here? The latest commit made the first line wrap

Reply:
The CLI is currently configured to 79 lines. I'm down to change that but it should probably be a separate PR.

Reply:
Ah oops. Then this change is probably _worse_ formatting

Reply:
Okay, I changed this up a bit